programs/bpf/c/sdk/bpf.mk

```diff
@@ -6,7 +6,7 @@ ifneq ($(V),1)
 _@ :=@
 endif
 
-SYSTEM_INC_DIRS ?= -isystem $(dir $(lastword $(MAKEFILE_LIST)))inc
+SYSTEM_INC_DIRS ?= -Isystem -I$(dir $(lastword $(MAKEFILE_LIST)))inc
```

Comment:
What clang are you using?  Mine (7.0) doesn’t need this, I think

Reply:
SYSTEM_INC_DIRS needs to be :=, not ?=.   I have a patch pending for this but didn’t land it today.  I can do it tomorrow if still needed

Reply:
I've always seen individual includes preceeded by `-I` (don't see any documentation on `-i`

Reply:
<sol_bpf.h> also works with `-I`

Reply:
Ok, scratch that, I think the inclusion issue I was seeing was due to the weird .d issue that I fixed.